python/cuml/fil/fil.pyx

```diff
@@ -539,7 +559,10 @@ class ForestInference(Base,
                                  algo='auto',
                                  threshold=0.5,
                                  storage_type='auto',
-                                 blocks_per_sm=0):
+                                 blocks_per_sm=0,
```

Comment:
Consider having a separate pull request that aggregates all named parameters into a single `**kwargs` parameter. This would make adding new named parameters much easier.